### DISCUSSION TITLE
Anyone else having drive mounting issues on wake up?

### DISCUSSION THREAD

**Discussion by mattenz** [marcone/teslausb#853] [General]
I think this is a Tesla issue and not a TeslaUSB issue as I've seen someone else mention the issue elsewhere who is using an SSD on its own.
Anyway, for a month or so now, if I wake the car up via opening a door or even opening the app, the drive won't be visible to the car and the dashcam icon with have the red X. However, if the car is woken by a request to start preconditioning then everything is fine and the drive mounts as it should.

When it doesn't mount, the only way to fix it is to either reboot the car or let it go back to sleep and then wake it backup with a precondition command. It seems like a really weird issue and I'm not sure what the difference would be between the way the car wakes up but I can make it do this every time. I'm hoping to find more people with the issue to see if there is some kind of common pattern that might be causing it.

**Comment by KrzysztofHajdamowicz** [comment]
Sounds like your Pi boots too slow and MCU/FSD computer timeouts.

When you wake the car by precondition USB ports are powered before MCU, allowing Pi to boot and offer drive before it's needed

**Reply by mattenz** [reply]
I guess that makes sense, but it didn’t used to do this. I also would have thought that the MCU should be able to mount the storage if it’s plugged in post-boot.

I’ll have to do a test in the morning with a standard USB drive and confirm if it’s happening with that too.
